opus/application/apps/ui/templates/ui/base.html

```diff
@@ -253,14 +253,14 @@ <h3 class='op-header-text'></h3>
 
         <!-- modal dialog for the gallery views -->
         <div id="galleryView" class="modal fade" role="dialog" tabbindex="-1" data-backdrop="false" data-keyboard="false" aria-labelledby="galleryViewLabel" aria-hidden="true">
-            <div class="modal-dialog modal-lg" role="document">
+            <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
```

Comment:
I thought we wanted to take this out for now

Reply:
my changes include a fix for this issue.